inspect/trace_api.go

```diff
@@ -61,6 +68,7 @@ func (tai *TraceAPIInspector) Inspect(ctx context.Context, inspectionCfg Inspect
 	if err != nil {
 		resultErr = multierror.Append(resultErr, fmt.Errorf("failed to get chain id: %w", err))
 		results.Indicators[IndicatorTraceAccessible] = ResultFailure
+		results.Indicators[IndicatorTraceSupported] = ResultFailure
 	} else {
 		results.Indicators[IndicatorTraceAccessible] = ResultSuccess
```

Comment:
Do we need to set "trace supported" as successful here?

Reply:
Did you mean "as failure"?

Reply:
Never mind - it makes sense